### PR TITLE
Adding core 6.1 to the suite

### PR DESCRIPTION
**Pull request by andreasohlund**
Plenty of manual tinkering :(

Wonder if there is a better way to do this

cc @timbussmann @dbelcham

### CODE REVIEW

**Comment by andreasohlund** [comment]
@Particular/nservicebus-maintainers please review and merge

File: src/Version_6_1/PubSub/EventHandler.cs
Comment (danielmarbach) [inline src/Version_6_1/PubSub/EventHandler.cs:17]: Does this need ` IHandleMessages<Version_6_1.Messages.MyEvent>`?

File: src/Version_6_1/SendReturn/SendReturnInitiator.cs
Comment (danielmarbach) [inline src/Version_6_1/SendReturn/SendReturnInitiator.cs:10]: Should we make this async as well to align with other of those extensions? Does it need `Task.Run`?

File: src/Version_6_1/PubSub/EventHandler.cs
Comment (andreasohlund) [reply src/Version_6_1/PubSub/EventHandler.cs:17]: Fixed

File: src/Version_6_1/SendReturn/SendReturnInitiator.cs
Comment (andreasohlund) [reply src/Version_6_1/SendReturn/SendReturnInitiator.cs:10]: Done

**Comment by andreasohlund** [comment]
@danielmarbach should be good to go once green

**Review by danielmarbach — APPROVED** [review]

**Pull request merged**
